garage/envs/mujoco/gather/embedded_viewer.py

```diff
@@ -56,15 +54,15 @@ def set_model(self, model):
             self.autoscale()
 
     def autoscale(self):
-        self.cam.lookat[0] = self.sim.stat.center[0]
-        self.cam.lookat[1] = self.sim.stat.center[1]
-        self.cam.lookat[2] = self.sim.stat.center[2]
-        self.cam.distance = 1.0 * self.sim.stat.extent
-        self.cam.camid = -1
+        self.cam.lookat[0] = self.model.stat.center[0]
+        self.cam.lookat[1] = self.model.stat.center[1]
+        self.cam.lookat[2] = self.model.stat.center[2]
+        self.cam.distance = 1.0 * self.model.stat.extent
+        self.cam.fixedcamid = -1
         self.cam.trackbodyid = -1
         if self.window:
             width, height = glfw.get_framebuffer_size(self.window)
-            functions.mjv_moveCamera(self.model, None, width, height, self.scn,
+            functions.mjv_moveCamera(self.model, 5, width, height, self.scn,
```

Comment:
I think writing a comment saying 5 == mjMOUSE_ZOOM would be helpful to understand this code

Reply:
Good idea, thanks!